pages/search.tsx

```diff
@@ -118,7 +119,7 @@ class Search extends Component<Props, State> {
 
     } catch (error) {
       console.log(error)
-      safeAlert('Search failed. Please check your internet connection and try again later.')
+      safeAlert(PV.pages.search.searchError)
```

Comment:
I'm going to move this from pages.search to pages.components.Alerts.searchError. It might get reused in the search results of another page.